src/verify-hostname.lisp

```diff
@@ -97,10 +97,11 @@
   (when dns-names
     (error 'unable-to-match-altnames))
   ;; TODO: we are matching only first CN
-  (alexandria:if-let ((cn (first (certificate-subject-common-names cert))))
-    (progn
-      (or (try-match-hostname cn hostname)
-          (error 'unable-to-match-common-name)))
+  (alexandria:if-let ((cns (certificate-subject-common-names cert)))
+    (or (some (lambda (cn)
+                (try-match-hostname cn hostname)) 
+              cns)
+      (error 'unable-to-match-common-name))
```

Comment:
we will need to align the OR branches

Reply:
Hmm, I can't see that misalignment?!

Reply:
I encountered it when reviewing a specific commit. The final version does not have it, so never-mind.

![image](https://user-images.githubusercontent.com/2444/103156920-ccc8c480-47be-11eb-83b7-d8570fb4ef32.png)